devices/linux/Files/main.py

```diff
@@ -1,42 +1,47 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-import subprocess 
+
+import base64
+import getpass
+import os
+import platform
+import re
+import subprocess
 import sys
+import uuid
+from shutil import copyfile
 
-def missing_dbus():
-    print("Cannot import the dbus module")
-    sys.exit(1)
-    
 try:
     import dbus
-except:
+except ImportError:
     if sys.version_info.major == 3:
-        missing_dbus()
+        print("Cannot import the dbus module")
+        sys.exit(1)
```

Comment:
Actually this script can still be helpful by generating a wpa2_supplicant config so we don't have to `exit()` here -- see #116 . If you like it you might add my code or do something similiar?!

Reply:
I removed this part of the code. Because we should this better discuss at your pull request and keep this pull request simple. I don't know otherwise how much time it needs to merge it.